code/game/atoms.dm

```diff
@@ -600,7 +600,7 @@
 /**
  * React to an EMP of the given severity
  *
- * Default behaviour is to send the [COMSIG_ATOM_EMP_ACT] signal
+ * Default behavior is to send the [COMSIG_ATOM_EMP_ACT] signal
```

Comment:
This is not a misspelling and goes against our [style guideline](https://github.com/tgstation/tgstation/blob/2f1ddbef0194c0a241f2f975a016effcae139f47/.github/guides/STYLE.md#things-that-do-not-matter)

Reply:
This whole PR is just this aside from a handful of actual in game text. i think it should be closed